tests/EventTest.php

```diff
@@ -165,6 +165,38 @@ public function testToArrayWithBreadcrumbs(): void
         $this->assertSame($breadcrumbs, $data['breadcrumbs']['values']);
     }
 
+    public function testToArrayWithMessageWithFormatted(): void
+    {
+        $expected = [
+            'message' => 'foo @bar',
+            'params' => ['@bar' => 'bar'],
+            'formatted' => 'foo bar',
+        ];
+
+        $event = new Event();
+        $event->setMessage('foo @bar', ['@bar' => 'bar'], 'foo bar');
+
+        $data = $event->toArray();
+
+        $this->assertArrayHasKey('message', $data);
+        $this->assertSame($expected, $data['message']);
+    }
+
+    public function testGetMessage(): void
+    {
+        $event = new Event();
+        $event->setMessage('foo @bar', ['@bar' => 'bar'], 'foo bar');
+        $this->assertSame('foo @bar', $event->getMessage());
```

Comment:
It doesn't change much in this case, but since we're comparing strings it makes no sense (in this case, in others it makes) to use strict comparison. It doesn't really change much, both are good but since in the codebase I believe that for the most times we did the same I've asked to change it to maintain consistency

Reply:
Hmm I use strict comparison everywhere as a habit, seems much safer.

Reply:
It is safer indeed, but in this case since we have typehinted the return type strictly comparing is not needed since we are sure that both arguments are strings

Reply:
Of course.. but I guess my assumption re: testing is that if code is refactored the typing could be wrong, so tests can be overly defensive.

Reply:
I could not think of any case where a wrong typehint while refactoring code even not using strict comparison could make the test not fail if both values are not equal (still, in this specific case). Anyway I get your point, if you prefer to leave it as is it's ok. The other requested changes are still needed of course